site/en/docs/extensions/whatsnew/index.md

```diff
@@ -15,6 +15,12 @@ tags:
 
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
+### Chrome 117: Expanded kill-URL protections on extension API navigations {: #expanded-kill-url }
```

Comment:
I realise this came from the doc, but I don't think we want to say kill-URL protections specifically - that refers to URLs like chrome://crash whereas this change affects all URLs:

```suggestion
### Chrome 117: Expanded URL protections on extension API navigations {: #expanded-kill-url }
```